backend/event-service/src/main/kotlin/com/ticketqueue/event/service/SeatService.kt

```
@@ -0,0 +1,102 @@
+     *
+     * Redis 장애 시 DB fallback으로 정상 응답을 보장한다.
+     */
+    fun getSeats(scheduleId: UUID): SeatDto.SeatsResponse {
+        val cacheKey = "$cacheKeyPrefix$scheduleId"
+
+        try {
+            val cached = redisTemplate.opsForValue().get(cacheKey)
+            if (cached != null) {
+                return objectMapper.convertValue(cached, SeatDto.SeatsResponse::class.java)
+            }
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)